### DISCUSSION TITLE
Error github "WARNING: UNPROTECTED PRIVATE KEY FILE!"

### DISCUSSION THREAD

**Discussion by MuehldorferAG** [requarks/wiki#5236] [Error / Bug Report] [answered]
Hello,

I have a problem with backup data to Github. All was working good but now I am getting this warning in Storage settings for GitHub:

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@ @ WARNING: UNPROTECTED PRIVATE KEY FILE! @ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@ Permissions 0744 for '/wiki/ssh' are too open. It is required that your private key files are NOT accessible by others. This private key will be ignored. Load key "/wiki/ssh": bad permissions git@github.com: Permission denied (publickey). fatal: Could not read from remote repository. Please make sure you have the correct access rights and the repository exists. error: Could not fetch origin

My settings are the same as before. In the process of debugging I created a new ssh key but it dint change anything.
I tried to change permission to 0600 but after that, I am losing access to that folder completely.

**Comment by NGPixel** [comment]
Either set the file persmissions to 0600 with the owner being the same user as wiki.js OR paste the contents of the private file directly into the Wiki.js admin instead (which is easier IMO).

**Reply by MuehldorferAG** [reply]
Hello,

I tried to set permissions of the key and folder to 0600 and the owner is node user because I have seen that node user is the user that has rights to access the wiki folder but after that, I am getting the error "permission denied". I really don't know what happened and why I can no longer use git backup.

Here is the content of the wiki docker container:
![wiki](https://user-images.githubusercontent.com/86592238/166146093-e7d8a1b7-707d-4cb3-bbf4-e06ec50771e2.png)

Here is wiki folder content with permissions:
![wiki2](https://user-images.githubusercontent.com/86592238/166146182-e299a5cb-8850-4026-af46-16bb56a2e9bc.png)

I put the key inside the ssh folder that I created. I am still getting that error. If you have any more suggestions I would be thankful for your support.

![wiki3](https://user-images.githubusercontent.com/86592238/166233685-0d46b6f6-0430-4ba0-bc16-19f61170754d.png)

![wiki4](https://user-images.githubusercontent.com/86592238/166233829-4571e259-0fb5-4514-b4fb-3ee4a9b57e7a.png)

**Comment by MuehldorferAG** [comment, marked as answer]
Hello,

After I made an update to the wiki today now it is working again.
I don't know if you made any changes, but thank you for your help. :)
